src/sparkx/Particle.py

```diff
@@ -430,7 +468,10 @@ def __initialize_from_array(
                 # It is important for JETSCAPE particles to compute pdg_valid
                 # here because the charge_from_pdg function depends on
                 # it.
-                self.pdg_valid = PDGID(self.pdg).is_valid
```

Comment:
Why did you change this? Is there something wrong with the old version?

Reply:
Yes. Imagine we have a custom input which does not provide PDG, then this would fail.

Reply:
OK